h/templates/activity/search.html.jinja2

```diff
@@ -200,12 +238,10 @@
   {% call sidebar(group.name, group.description) %}
     <section class="search-result-sidebar__section">
       <dl>
-        {% if stats.annotation_count  %}
-          <dt class="search-result-sidebar__dt">
-            {% trans %}Shared annotations:{% endtrans %}
```

Comment:
I'm guessing the "Shared annotations" text previously used was chosen to indicate that the count did not include private annotations. I'm not sure if that was in fact even true in the first place - I think here what we're actually showing is "Shared annotations by other people plus your private annotations". Calling it just "Annotations" seems cleaner.